src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -36,7 +38,57 @@
 JOBS_MISSING_ERR = "No valid job IDs provided"
 
 CELLS_NOT_PROVIDED_ERR = "cell_id_list not provided"
-DOES_NOT_EXIST = "does_not_exist"
+
+
+class OutputStateErrMsg(Enum):
+    """
+    For errors that go into the STATUS response.
+    Enum mapping from error names to formattable error messages.
+    The first formatting argument of each error message must be the job ID
+    """
+
+    NOT_FOUND = "Cannot find job with ID %s"
+    NOT_UPDATED = "Job with ID %s has not been updated since ts %d"
+    QUERY_EE2_STATES = "A Job.query_ee2_states error occurred for job with ID %s: %s"
+    CANCEL = "An EE2.cancel_job error occurred for job with ID %s: %s"
+
+    def gen_err_msg(self, its: List[Union[str, int, Iterable]]):
```

Comment:
To be used with `jm.add_errors_to_results`. Returns iterable that will return errors formatted with appropriate args (i.e., `error_ids`, other error msg information)

Reply:
How are you handling error messages that come back from ee2? Just passing them through as-is?

Reply:
I believe that was the original behavior which I preserved.